components/footers/LinksList.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <ul>
+    <li v-for="el in elements" :key="el.label">
+      <nuxt-link
+        v-if="el.linkType==='nuxtLink'"
+        class="footer-link"
+        :to="el.url"
+      >
+        {{ el.label }}
+      </nuxt-link>
+      <a
+        v-else
```

Comment:
For implementing `Cta` components, we also use `a` or `nuxt-link` ones. We decide when to use one or another, and also what value `target` should be by [inspecting the URL](https://github.com/Qiskit/qiskit.org/blob/master/components/ctas/Cta.vue) and using [dynamic components](https://vuejs.org/v2/guide/components.html#Dynamic-Components).

This way, if the link is something like `/advocates`, a `nuxt-link` component is used with `target="_self"`. If the URL looks like `https://something.com/external`, then we use an `a` tag with `target="_blank"` and `rel="noopener"`.

However, qiskit.org hosts two static sites inside: API documentation and the textbook. These require using the `a` tag without opening a new window and no `rel` values.

We solve this with a flag, as you do with `nuxtLink`/`aLink` but instead of being related to the underlaying implementation, it is related with the nature of the content and so, it is called `is-static`.

My recommendation is to adhere to the same solution and refactor the solution in a follow-up. If you prefer not to alter your solution right now, that's fine also. Just remember to open another issue for consolidating both strategies and refactor them. It is up to you.

Reply:
I have created [this issue](https://github.com/Qiskit/qiskit.org/issues/405) to unified both solutions